### ISSUE TITLE
SetPersonFrame bug

### ISSUE THREAD

**Issue by Radnen** [spheredev/neosphere#21]
SetPersonFrame should honor Sphere's fact that if you set a frame greater than the max frame, it just modulus-wraps it.

In my game blockman, when you pick up an item, this fails:

``` javascript
SetPersonFrame(this.name, GetPersonFrame("player"));
```

It sets the frame of the held Entity to the same as Blockman, but the held entity only has 2 frames whereas Blockman has 4+. In Sphere we expect to see the other entity just loop through it 2 frames, even if it was told to go to frame '4', for instance.

**Comment by fatcerberus** [comment]
It throws an exception, right?  I remember being iffy about that the last time I worked with this component.  I didn't study Sphere's implementation particularly closely for this API, and obviously I got it wrong.

It's a quick enough fix, I'll put it on the agenda for 1.0.10.  Looks like minisphere 1.0 still has a bit of life left yet. :-)  Honestly I want to get as many small issues like this as possible out of the way first before committing to 1.1 anyway (which is why I chose to release the FollowPerson update as 1.0.9, incidentally).
